engine_test.go

```diff
@@ -148,6 +148,11 @@ output: []`),
 }
 
 func TestE2E(t *testing.T) {
+	if testing.Short() {
+		// This test requires pulling containers; it must not be used when count > 2
+		t.Skip("skipping e2e test in short mode")
```

Comment:
I kinda like the *comment* better than the actual "skip" message. Is the problem "e2e" tests, or "pulling containers"? (And if "count > 2" is actually part of the issue, is that something the "skip" should/can be sensitive to?)

Reply:
I removed the comment and added the container info to the skip message. How is that?
There is no way to check the count value in-test that I can see.

I checked and it appears to scale linearly, but it only takes a few runs to exceed one minute. 1000 runs would take almost 4 hours.